### DISCUSSION TITLE
post/kakaopay-payment-patent/

### DISCUSSION THREAD

**Discussion by giscus** [kakaopay/techlog-comments#46] [Announcements]
# post/kakaopay-payment-patent/

카카오페이 오프라인 결제 특허의 기획부터 등록 과정을 Best Practice인 PC방 결제와 엮어서 소개합니다.

https://tech.kakaopay.com/post/kakaopay-payment-patent/

**Comment by pay-dan-dy** [comment]
서비스화한 기능을 특허로 등록하기까지의 겅험을 공유해 주셔서 감사합니다! 재밌게 잘 읽었어요~

**Comment by marierose1980** [comment]
게토를 쓰는 피시방에 무선 카드 단말기를 들였다는 기사를 봤는데 앉은 자리에서 카카오페이로 화면에 뜨는 QR 코드를 인식해서 결제하면 되는데 "굳이?"하는 생각이 들었습니다.
